pipeline/src/test_util.rs

```diff
@@ -539,3 +539,27 @@ pub fn run_reparse_test_with_blacklist(file: &str, blacklist: &[&str]) {
         .compute_optimized_pil()
         .unwrap();
 }
+
+// TODO: Add #[cfg(feature = "stwo")] to conditionally compile this function. now when I add it, this code is disabled. how to eable by default?
```

Comment:
Add noop version when the feature is not active and remove comment

Reply:
Done